app/main.py

```diff
@@ -1,3 +1,59 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: object) -> "Distance":
```

Comment:
Try to not use "object" as type for arguments, because it is not a good practice. Fix it everywhere
```suggestion
    def __add__(self, other: int | float| Distance) -> "Distance":
```